drive/driveimpl/drive_test.go

```diff
@@ -78,19 +82,23 @@ func TestFileManipulation(t *testing.T) {
 
 	s.addRemote(remote1)
 	s.addShare(remote1, share11, drive.PermissionReadWrite)
-	s.writeFile("writing file to read/write remote should succeed", remote1, share11, file111, "hello world", true)
+	s.writeFile("writing file to read/write remote should succeed", remote1, share11, file111, contents, true)
 	s.checkFileStatus(remote1, share11, file111)
 	s.checkFileContents(remote1, share11, file111)
+	// Despite this being an HTML file, Content-Type should be text/plain to
+	// prevent user agents from treating this as an HTML page. This prevents
+	// Taildrive from being used to distribute malicious JavaScript.
+	s.checkContentType(remote1, share11, file111, "text/plain")
```

Comment:
I verified that without the fix, this fails because the Content-Type is `text/html`.